### PR TITLE
Add secret to grafana-agent to store logging write credentials

### PR DESCRIPTION
**Pull request by marieroque**
towards https://github.com/giantswarm/giantswarm/issues/28486

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

File: helm/observability-bundle/values.yaml [hunk 1]
```diff
@@ -115,3 +115,6 @@ apps:
       - kind: configMap
         name: "{{ $.Values.clusterID }}-grafana-agent-config"
         namespace: "{{ $.Release.Namespace }}"
+      - kind: secret
+        name: "{{ $.Values.clusterID }}-grafana-agent-secret"
+        namespace: kube-system
```
Comment (QuentinBisson) [inline helm/observability-bundle/values.yaml:120]: Why kube-system?

Comment (marieroque) [reply helm/observability-bundle/values.yaml:120]: Because I need this secret where the agent is running to use `envFrom`

Comment (QuentinBisson) [reply helm/observability-bundle/values.yaml:120]: So { $.Release.Namespace }} right?

Comment (marieroque) [reply helm/observability-bundle/values.yaml:120]: Humm it's not what I see.
{ $.Release.Namespace }} => namespace where the app is deployed, so `giantswarm`

Comment (QuentinBisson) [reply helm/observability-bundle/values.yaml:120]: Well yes but then that will never work on WCs :)

Comment (QuentinBisson) [reply helm/observability-bundle/values.yaml:120]: Beause this secret would be created on the MC cluster

**Comment by marieroque** [comment]
CHANGELOG already up to date in the previous PR already merged

**Review by QuantumEnigmaa — APPROVED** [review]

**Pull request merged**